R/metab.bookkeep.R

```diff
@@ -64,7 +64,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
   
 	if("datetime"%in%names(mb.args)){ # check to see if datetime is in the ... args
 		datetime <- mb.args$datetime # extract datetime
-		freq <- calc.freq(datetime) # calculate sampling frequency from datetime
+		freq <- calc.freq(datetime) - 1 # calculate sampling frequency from datetime
 		if(nobs!=freq){ # nobs and freq should agree, if they don't issue a warning
```

Comment:
This does need to be fixed then, as frequency is not == to nobs necessarily. Should be `(nobs +1) != freq ` I believe.